css-typed-om/Overview.bs

```diff
@@ -784,6 +787,72 @@ The following are the arithmetic operations you can perform on dimensions:
         is set to |this|.
 </div>
 
+<div algorithm="CSSNumericValue.min()">
+    The <dfn for=CSSNumericValue method>min(...|values|)</dfn> method,
+    when called on a {{CSSNumericValue}} |this|,
+    must perform the following steps:
+
+    1. Replace each [=list/item=] of |values|
+        with the result of [=rectifying a numberish value=] for the [=list/item=].
+
+    2. Prepend |this| to |values|.
+
+    3. If there are at least two different [=strong types=]
+        among the [=list/items=] in |values|,
+        [=throw=] a {{TypeError}}.
+
+    4. If all of the [=list/items=] in |values| are {{CSSUnitValue}}s
+        and have the same {{CSSUnitValue/unit}},
+        return a new {{CSSUnitValue}}
+        whose {{CSSUnitValue/unit}} internal slot is set to |this|’s {{CSSUnitValue/unit}} internal slot,
+        and {{CSSUnitValue/value}} internal slot is set to the minimum value of
+        the {{CSSUnitValue/value}} internal slots
+        of the [=list/items=] in |values|.
+
+    5. If |this| is a {{CSSMathMin}} object,
```

Comment:
And I've done that now, so instead be consistent with text of the new functions. ^_^